tsconfig.json

```diff
@@ -21,5 +21,5 @@
     "strictNullChecks": true,
     "target": "es2017"
   },
-  "files": ["./src/index.ts"]
+  "exclude": ["./dist"]
```

Comment:
Hmmm. It will cause tests to be included in the build output. Should we have two separate `tsconfig` files?